fs-utils.go

```diff
@@ -21,45 +21,36 @@ import (
 	"unicode/utf8"
 )
 
-// validBucket regexp.
-var validBucket = regexp.MustCompile(`^[a-z0-9][a-z0-9\.\-]{1,61}[a-z0-9]$`)
+// validVolname regexp.
+var validVolname = regexp.MustCompile(`^.{3,63}$`)
```

Comment:
I think we need to have more validation here ie name with `/`, '\'  and other special chars are also getting passed.  I see Windows doesn't allow `/ \ : * ? " < > |` and linux doesn't allow `/`